softwareComponents/rofiHalEsp32/src/peripherals/dynamixel/packet.hpp

```diff
@@ -232,7 +232,7 @@ inline uint16_t update_crc(uint16_t crc_accum, const uint8_t *data_blk_ptr, int
 
     for(j = 0; j < data_blk_size; j++)
     {
-        i = ((uint16_t)(crc_accum >> 8) ^ data_blk_ptr[j]) & 0xFF;
+        i = (uint16_t(crc_accum >> 8) ^ data_blk_ptr[j]) & 0xFF;
```

Comment:
Změna je v pohodě, ale proč se to jednou posouvá doprava a podruhé doprava? Vypadá to jako bug...

Reply:
Tohle je copy & paste kód z datasheetu. A záměrně ho nechávám co nejméně změněny.